web/src/main/java/com/epam/esm/web/converter/GiftCertificateConverter.kt

```diff
@@ -0,0 +1,58 @@
+package com.epam.esm.web.converter
+
+import com.epam.esm.GiftCertificateOuterClass
+import com.epam.esm.TagList
+import com.epam.esm.model.entity.GiftCertificate
+import com.epam.esm.model.entity.Tag
+import org.springframework.stereotype.Component
+
+@Component
+class GiftCertificateConverter(
+    private val dateConverter: DateConverter,
+    private val tagConverter: TagConverter
+) {
+    fun entityToProto(
+        giftCertificate: GiftCertificate
+    ): GiftCertificateOuterClass.GiftCertificate {
+        val tagListOfProto =
+            giftCertificate.tagList.map { tag ->
```

Comment:
it would be better to move .map to a new line
```suggestion
            giftCertificate.tagList
                .map { tagConverter.tagToProto(it) }
                .toList()
```

Reply:
✅